### ISSUE TITLE
Works only if player is in creative mode. 

### ISSUE THREAD

**Issue by SlitherySnekks** [LCLPYT/illusory-walls#3]
Was planning on using it in a modpack, but tool only works in creative making it somewhat pointless.

**Comment by LCLPYT** [comment]
Yes indeed, sorry for replying so late. I currently don't know how this should be balanced. With the Staff of Illusion, you can just mark **any block** as illusory wall. I don't know how I would balance this, so that bedrock etc. cannot be marked.

I could add durability to the staff and remove some depending on the hardness of the block to convert.

Do you have any suggestions maybe?

**Comment by SlitherySnekks** [comment]
Hardness of the block is a good idea... Maybe limiting the number of ghost blocks per player in general. I don't see having more than 10 blocks per player on a server being a good idea...

Could make a tiered staff similar to how the staff of building is. Better staff, more blocks you can use it on. 
Maybe add long cool-downs to low tier staffs and higher or adjustable on higher tier ones. 

Admittedly I haven't used the mod since i was playing on a survival server. So maybe you already have a block limit in the config, wouldn't know.

**Comment by SlitherySnekks** [comment]
Maybe a simpler solution is having 3-4 staff items (progressively more difficult to craft), and just have each have an adjustable block count, and cool-down in a config. So an admin can set the limit to their liking on each staff, or completely disable them. 

Just makes it simpler than trying to balance for survival or balance for pvp or a mass server etc etc, make it adjustable and make it an end user problem if its op or not.

**Comment by LCLPYT** [comment]
Good idea, I'll try to implement something like that soon 👍
